### PR TITLE
Update wascc-host to 0.5.0

### PR DESCRIPTION
**Pull request by bketelsen**

### CODE REVIEW

File: crates/wascc-provider/src/lib.rs [hunk 1]
```diff
@@ -108,7 +108,7 @@ impl Provider for WasccProvider {
             .get(ACTOR_PUBLIC_KEY)
             .map(|a| a.to_string())
             .unwrap_or_else(|| "".into());
-
+        debug!("{:?}", pubkey);
```
Comment (thomastaylor312) [inline crates/wascc-provider/src/lib.rs]: Did you want this to be echoed during debug?

Comment (bketelsen) [reply crates/wascc-provider/src/lib.rs]: removed it, but it is really nice seeing the public key of the actor as you go through debugging

**Comment by bketelsen** [comment]
fixes #30

**Comment by bketelsen** [comment]
removing tests - captured in #36

**Pull request merged**